### PR TITLE
Improve lint tests

### PR DESCRIPTION
**Pull request by 153957**
Replace flake8/black by ruff for code style and formatting.
Add typos for spell checking.

Fix Python tests cleanup by disabling multiprocessing.

### CODE REVIEW

File: manage.py
Comment (davidfokkema) [inline manage.py:1]: Black would change this from single quotes to double quotes... does ruff have very different style rules?

File: publicdb/analysissessions/urls.py
Comment (davidfokkema) [inline publicdb/analysissessions/urls.py:1]: Here, too. Double quotes to single quotes?

**Review by davidfokkema — COMMENTED** [review]
I've heard great things about ruff, nice! The quote situation does surprise me, though. Lots of double quotes are changed to single quotes. Is this ruff?

File: manage.py
Comment (153957) [reply manage.py:1]: Black always prefers/uses double quotes unless you tell it to skip quote normalization, which I did when I applied black.
Ruff allows it to be configured, to tell it which 
https://github.com/HiSPARC/publicdb/blob/testing-updates/pyproject.toml#L83-L87
I prefer single quotes, so I chose that.

**Review by davidfokkema — APPROVED** [review]

**Pull request merged**